src/index.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css2?family=Barlow+Semi+Condensed:wght@500;600;700&family=Cairo:wght@200;300;400;500;600;700&family=Goudy+Bookletter+1911&display=swap');
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+  font-family: 'Barlow Semi Condensed', sans-serif;
+}
+
+.link.one {
+  margin-left: auto;
+}
+
+.link {
+  font-size: 20px;
+  text-decoration: none;
+  color: #222831;
+  padding: 0.4rem 0.4rem;
+  border-radius: 3px;
+}
+
+.link:hover {
+  text-decoration: underline;
+}
+
+.link.active {
+  color: #ffd369;
+  text-decoration: underline;
+  background-color: #393e4665;
+  filter: drop-shadow(0.25rem 0.25rem 0.5rem #222831);
+}
+
+.home {
+  scroll-behavior: smooth;
+}
+
+.racks {
+  background-color: #222831;
+}
+
+.header {
+  color: #ffd369;
+  font-family: 'Barlow Semi Condensed', sans-serif;
+  margin: 0 auto;
+  padding-top: 3rem;
+  font-size: 72px;
+  width: 75rem;
+}
+
+.movies-rack,
+.series-rack {
+  padding: 5rem 0 0;
+}
+
+.series-rack {
+  padding-bottom: 5rem;
+}
+
+.rack-head {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+  line-height: 5rem;
+  width: 75rem;
+  margin: 0 auto 1rem;
+}
+
+.rack-head h1 {
+  font-size: 60px;
+  color: #ffd369;
+  margin: 1rem 1rem 0 0;
+}
+
+.rack-head h2 {
+  font-family: 'Barlow Semi Condensed', sans-serif;
+  line-height: 0;
+  padding-top: 1rem;
+  font-size: 42px;
+  font-weight: 600;
+  color: #eee;
+}
+
+.arrow {
+  border-top: 5px solid #eee;
+  border-right: 5px solid #eee;
+  height: 1.3rem;
+  width: 1.3rem;
+  transform: rotate(45deg);
+  align-self: flex-end;
+  margin: 0 0 1.3rem 0.5rem;
+}
+
+.page-link {
+  text-decoration: none;
+  color: #eee;
+  font-size: 18px;
+  margin-left: 29rem;
+  align-self: flex-end;
+  line-height: 25px;
+}
+
+.page-link:hover {
+  text-decoration: underline;
+  color: #ffd369;
+}
```

Comment:
- [ ] Your project is very impressive you implement almost everything needed for this project and went beyond the requirement :clap: But you forgot to follow the project [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) based on Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio). :cry:  Please make sure to follow the design including Typography (font face, size, and weight) and Layout (composition and space between elements) as required for this project. 
- [ ] Please make sure that you have your home and details page following the design guidelines and also the filter as defined in the design below :arrow_down: . As a developer it is very important to pay attention to details :smile: